cadcutils/dev_requirements.txt

```diff
@@ -0,0 +1,8 @@
+numpy==1.12.0b1
+astropy==1.2.1
+cadcutils==1.0
```

Comment:
Which one is vofs? 
I'm assuming this repo, so i'll send a pull request there. 
https://github.com/opencadc/vostools